lib/matplotlib/markers.py

```diff
@@ -854,65 +785,48 @@ def _set_x(self):
         self._path = self._x_path
 
     _plus_filled_path = Path(
-        [(1/3, 0), (2/3, 0), (2/3, 1/3), (1, 1/3), (1, 2/3), (2/3, 2/3),
-         (2/3, 1), (1/3, 1), (1/3, 2/3), (0, 2/3), (0, 1/3), (1/3, 1/3),
-         (1/3, 0)], closed=True)
+        np.array([(-1, -3), (+1, -3), (+1, -1), (+3, -1), (+3, +1), (+1, +1),
+                  (+1, +3), (-1, +3), (-1, +1), (-3, +1), (-3, -1), (-1, -1),
+                  (0, 0)]) / 6, closed=True)
```

Comment:
```suggestion
                  (-1, -3)]) / 6, closed=True)
```
I think the last value is ignored, but still it's better to write the correct position.

Reply:
Actually I tend to think it's better to write something nonsensical for closepolys, to make it more clear that it's a special node... (ditto below)

Reply:
IMHO ignoring the last value for `closed` is an implementation detail (we could have required it to be identical with the first node). It's still better to convey the intended location of the point. Then readers don't have to know about `closed` and still understand what is happening.

If you really want a nonsensical value though, you should use `(np.nan, np.nan)`. `(0, 0)` is much to sensical here 😝 .

Reply:
Sure, I won't insist here.